### ISSUE TITLE
Permission denied error : windows 10 can't create python 3.5 environment

### ISSUE THREAD

**Issue by Tahsin-Mayeesha** [conda/conda#4645]
error message looks like this. I tried without writing that anaconda in the end too. Note that I've successfully created and managed py35 environment when I was using windows 8 2/3 days ago. I've downloaded anaconda yesterday so the trouble is probably in the latest version.

```
$ C:\ProgramData\Anaconda2\Scripts\conda-script.py create -n py35 python=3.5 anaconda`



    Traceback (most recent call last):
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\exceptions.py", line 617, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\cli\main.py", line 137, in _main
        exit_code = args.func(args, p)
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\cli\main_create.py", line 68, in execute
        install(args, parser, 'create')
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\cli\install.py", line 210, in install
        unknown=index_args['unknown'], prefix=prefix)
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\core\index.py", line 120, in get_index
        index = fetch_index(channel_priority_map, use_cache=use_cache)
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\core\index.py", line 445, in fetch_index
        repodatas = _collect_repodatas(use_cache, urls)
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\core\index.py", line 433, in _collect_repodatas
        repodatas = _collect_repodatas_serial(use_cache, urls)
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\core\index.py", line 401, in _collect_repodatas_serial
        for url in urls]
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\core\index.py", line 141, in func
        res = f(*args, **kwargs)
      File "C:\ProgramData\Anaconda2\lib\site-packages\conda\core\index.py", line 391, in fetch_repodata
        with open(cache_path, 'w') as fo:
    IOError: [Errno 13] Permission denied: u'C:\\ProgramData\\Anaconda2\\pkgs\\cache\\2116b818.json'

```

**Comment by Tahsin-Mayeesha** [comment]
Turns out everything works if I do ```conda create``` from anaconda prompt.

**Comment by RookyKestral001** [comment]
Right-click anaconda prompt. Run as administrator.

**Comment by kalefranz** [comment]
Permissions errors and multi-user/read-only installs have been fixed in later versions of conda 4.3.  Also fixed with transactions in 4.3 is conda removing itself during a failed install/update operation.
